web/cobrands/fixmystreet.com/base.scss

```diff
@@ -369,6 +369,21 @@ html.lazyload .js-lazyload {
     background: none;
     margin: 0;
   }
+
+  // Applies text-decoration underline only to links that are in areas where there is a lot of text, so they could be easily missed by someone with visual impairments.
+  a {
+    text-decoration: underline;
```

Comment:
Happy for this to be .com only at the mo, yeah. Doing it for sidebar pages seems good to me, the admin looks fine I think.